resources/planwise/plpgsql/fix-isolated.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE FUNCTION insert_way (class_id integer, speed integer, source bigint, target bigint)
+RETURNS integer AS $$
+DECLARE
+  source_node RECORD;
+  target_node RECORD;
+  geom GEOMETRY;
+  length_m float;
+  new_gid BIGINT;
+BEGIN
+  SELECT * FROM ways_vertices_pgr WHERE id = source INTO source_node;
+  SELECT * FROM ways_vertices_pgr WHERE id = target INTO target_node;
+
+  geom := ST_MakeLine(ARRAY[source_node.the_geom, target_node.the_geom]);
+  length_m := ST_Length(geom::geography);
+
+  INSERT INTO ways
+         (class_id, length, length_m, source, target, x1, y1, x2, y2,
+          cost, one_way, maxspeed_forward, the_geom)
+         VALUES (class_id, ST_Length(geom), length_m, source, target,
+                 ST_X(source_node.the_geom), ST_Y(source_node.the_geom),
+                 ST_X(target_node.the_geom), ST_Y(target_node.the_geom),
+                 ST_Length(geom), 0, speed, geom)
```

Comment:
If we run this function independently, then it'd make sense to consider the traffic factor when inserting the cost here.